packages/mui-material/src/Rating/Rating.js

```diff
@@ -156,7 +156,7 @@ const RatingIcon = styled('span', {
     transform: 'scale(1.2)',
   }),
   ...(ownerState.iconEmpty && {
-    color: (theme.vars || theme).palette.action.disabled,
```

Comment:
Maybe we should change what the `action.disabled` color means instead of just changing it locally here in the Rating component? It's good to keep using a token we have for disabled stuff. And I'm assuming that if the contrast is not optimal here, it won't be in other places, too.

Reply:
It makes sense! Just a quick question, shouldn't it be the `action.active` instead? This token is not meant for the disabled stars, it's for the ones that aren't selected. That would solve the problem, since the contrast ratio that we need is 3:1 AA, what means that we'll need it to be at least rgba( 0, 0, 0, 0.42), and the `action.active` is rgba( 0, 0, 0, 0.54)

Reply:
Sounds like the solution! 🤙